lms/static/js/spec/edxnotes/plugins/store_error_handler_spec.js

```diff
@@ -2,30 +2,35 @@ define([
     'jquery', 'underscore', 'annotator_1.2.9',
     'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers',
     'js/spec/edxnotes/helpers',
-    'js/edxnotes/views/notes_factory'
-], function($, _, Annotator, AjaxHelpers, Helpers, NotesFactory) {
+    'js/edxnotes/views/notes_factory',
+    'js/edxnotes/utils/notes_collector'
+], function($, _, Annotator, AjaxHelpers, Helpers, NotesFactory, NotesCollector) {
     'use strict';
     describe('Store Error Handler Custom Message', function() {
         beforeEach(function() {
             spyOn(Annotator, 'showNotification');
             loadFixtures('js/fixtures/edxnotes/edxnotes_wrapper.html');
-            this.wrapper = document.getElementById('edx-notes-wrapper-123');
+            NotesCollector.cleanup();
```

Comment:
Nit: it would be cleaner if the clean up was done in the ``afterEach``. If it is needed in the ``beforeEach`` then some other tests are not cleaning up after themselves.